deployments/helm/nvidia-device-plugin/templates/daemonset.yml

```diff
@@ -194,7 +194,7 @@ spec:
       volumes:
         - name: device-plugin
           hostPath:
-            path: /var/lib/kubelet/device-plugins
+            path: {{ .Values.hostDir }}/device-plugins
```

Comment:
As a matter of interest, why is the full directory not configuratble and just the kubelet one? Should other paths such as the kubelet socket depend on this too?

Reply:
Greetings @elezar,
I submitted this based on the scope of our project. Thanks